src/plugins/camera/camera_definition_files/generated/camera_definition_files.cpp

```diff
@@ -0,0 +1,13791 @@
+/* Automatic generated file. DO NOT EDIT.
+   The file was generated by cpp_rsc program
+   https://github.com/orex/cpp_rsc
+   Resource file name: /tmp/tmp.FanET5d6hS/camera_definition_config.rsc
+   This is a source file*/
+
+#include "camera_definition_files.h"
+
+  /* Variable cgoetxml from file /home/jones/Documents/px4/sdk/mavsdk/tools/../src/plugins/camera/camera_definition_files/cgoet.xml */
```

Comment:
It would be better if the path doesn't end up in here :thinking:

Reply:
Yeah I was not a big fan of those comments, but I did not really feel like patching it :sweat_smile:. Anyway, I forked it and removed the comments!

Reply:
Oh really, nice :)